src/assets/style/custom.css

```diff
@@ -16,4 +16,7 @@
 body {
     background-color: #f5f6fa;
     color: #343a40;
+}
+.pointer-hand{
```

Comment:
What we are talking about is the consistency on the CSS class definition, we are requesting is to change `.pointer-hand{` to `.pointer-hand {` @jgilbertcastro.